subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -184,6 +186,36 @@ config BT_FAST_PAIR_STORAGE_INTEGRATION_INIT_PRIORITY
 
 endif # BT_FAST_PAIR_STORAGE_INTEGRATION
 
+config BT_FAST_PAIR_BOND_MANAGER
+	bool "Fast Pair bond management functionality [EXPERIMENTAL]"
+	default y
+	depends on !BT_FAST_PAIR_FMDN
```

Comment:
Shouldn't we introduce `BT_FAST_PAIR_USE_CASE_UNSUPPORTED_BOND_MANAGER` instead of depending on `!BT_FAST_PAIR_FMDN`? That could improve consistency with approach taken for other Kconfigs. @kapi-no what do you think?

Reply:
You are right. Seems that I overlooked it.